gcc/build/build.sh

```diff
@@ -142,16 +138,6 @@ make -j$(nproc)
 make ${INSTALL_TARGET}
 popd
 
-if [[ ! -z "${BINUTILS_VERSION}" ]]; then
-    # Work around insane in-tree built ld issue
-    for bindir in ${STAGING_DIR}/{,x86_64-linux-gnu/}bin
```

Comment:
This was needed a long while ago to get older GCCs to build. I _try_ to keep older builds building too with the same script (hence the patches etc). I'll do some testing to see if I'm able to build a few representative older versions too.

Reply:
Actually I probably didn't need to get rid of this bit. I assumed it was only related to the combined tree build (based on the comment about in-tree) but if it serves additional purposes it probably doesn't hurt to keep it.